packages/ecs-agent/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "ecs-agent"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
+
+# ECS agent
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/aws/amazon-ecs-agent/archive/v1.41.0.tar.gz"
+sha512 = "d53d1a47eb41d39ffd541b6e416b42276f1d2207fdf5f031be173a8737655dab71ca0c88732716b6fee0761f7fede474cee08a21fd2db5d33b460d6aa547630e"
+
+# Pause container
+
+# awsvpc CNI plugin
+
+# Other CNI plugin?
+
+#[[package.metadata.build-package.external-files]]
+#url = "https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-v1.40.0.tar"
+#sha512 = "1f22be420dc42aeefa249ef56fe4a594159c97d65ff65ee777fa7823a539e35b917c40d7679cccf3401b90bba045b53a6f3b17c53a034392295c935cc1c59cfa"
+
```

Comment:
Should we clean these up?

Reply:
Yes, though I'm going to change it to be a more clear TODO comment.  The CNI plugins are included upstream via git submodules but not included in the tar, so they'll need to be brought in as other sources.  I'm not planning to actually *do* that as part of this PR, but I want to make sure there's a note marking the remaining work.

Reply:
Done